.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 target/
 *.iml
+appstore-metadata-service-tests/build/spock-reports/
```

Comment:
it might be worth to just put the whole build/ dir here

Reply:
changed